test/integration/QuickstartTest.kt

```diff
@@ -42,13 +49,13 @@ class QuickstartTest: IntegrationTest() {
 
     @Test
     fun Quickstart() {
-        studioTest(composeRule) {
-            connectToTypeDB(composeRule, DB_ADDRESS)
+        studioTestWithRunner(composeRule) { address ->
+            connectToTypeDB(composeRule, address)
             createDatabase(composeRule, dbName = testID)
-            cloneAndOpenProject(composeRule, source = TQL_DATA_PATH, destination = testID)
-            writeSchemaInteractively(composeRule, dbName = testID, SCHEMA_FILE_NAME)
-            writeDataInteractively(composeRule, dbName = testID, DATA_FILE_NAME)
-            verifyDataWrite(composeRule, dbName = testID, "$testID/$QUERY_FILE_NAME")
+            cloneAndOpenProject(composeRule, source = Utils.TQL_DATA_PATH, destination = testID)
+            writeSchemaInteractively(composeRule, dbName = testID, Utils.SCHEMA_FILE_NAME)
+            writeDataInteractively(composeRule, dbName = testID, Utils.DATA_FILE_NAME)
+            verifyDataWrite(composeRule, address, dbName = testID, "$testID/${Utils.QUERY_FILE_NAME}")
```

Comment:
verifyDataWrite connects to TypeDB outside of Studio, so we need to pass the address to it as well now that we can't know the address in advance.